doc/level1.rst

```diff
@@ -416,72 +422,91 @@ Recommendations for single-recipient messages
+more than a month older than ``gossip_key_timestamp``, set
+``target-keys[to-addr]`` to ``gossip_key`` and set
+``preliminary-recommendation`` to ``discourage`` and skip to the
+:ref:`final-recommendation-phase`.
 
-Otherwise, if either ``public_key`` is null, or
-``autocrypt_timestamp`` is more than a month older than
-``gossip_key_timestamp``, the ``gossip_key`` is used with a
-preliminary recommendation of ``discourage``.
+Otherwise, set ``target-keys[to-addr]`` to ``public_key``.
 
+If ``autocrypt_timestamp`` is more than a month older than
+``last_seen``, set ``preliminary-recommendation`` to ``discourage``.
 
-Otherwise, the ``public_key`` is used, with the following preliminary
-recommendation:
-- If ``autocrypt_timestamp`` is more than a month older than
-  ``last_seen``, the preliminary recommendation is ``discourage``.
-- Otherwise, the preliminary recommendation is ``available``.
+Otherwise, set ``preliminary-recommendation`` to ``available``.
 
-The second phase turns on encryption by setting the recommendation
-to ``encrypt`` in two scenarios:
+.. _final-recommendation-phase:
 
-- If the preliminary recommendation is either ``available`` or
+
+Deciding to Encrypt by Default
+______________________________
+
+The final phase turns on encryption by setting ``ui-recommendation`` to
+``encrypt`` in two scenarios:
+
+- If ``preliminary-recommendation`` is either ``available`` or
   ``discourage``, and the message is composed as a reply to an
-  encrypted message
-- If the preliminary recommendation is ``available`` and
-  both ``peers[to-addr].prefer_encrypt`` and
-   ``accounts[from-addr].prefer_encrypt`` are ``mutual``
+  encrypted message, or
+- If the ``preliminary-recommendation`` is ``available`` and both
+  ``peers[to-addr].prefer_encrypt`` and
+  ``accounts[from-addr].prefer_encrypt`` are ``mutual``.
 
-Otherwise, the recommendation is set to the preliminary recommendation.
+Otherwise, the ``ui-recommendation`` is set to
+``preliminary-recommendation``.
 
 Recommendations for messages to multiple addresses
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For level 1 MUAs, the Autocrypt recommendation for a message
-composed to multiple recipients is derived from the recommendations
-for each recipient individually:
+For level 1 MUAs, the Autocrypt recommendation for a message composed
+to multiple recipients, we derive the message's recommendation from
+the recommendations for each recipient individually.
+
+The aggregate ``target-keys`` for the message is the merge of all
+recipient ``target-keys``.
+
+The aggregate ``ui-recommendation`` for the message is derived as
+follows:
 
-1. If any recipient has a recommendation of ``disable``, then the
-   message recommendation is ``disable``.
+1. If any recipient has a ``ui-recommendation`` of ``disable``, then
+   the message's ``ui-recommendation`` is ``disable``.
 2. If the message being composed is a reply to an encrypted message,
```

Comment:
This is redundant, replies will always be "disable" or "encrypt". might be fine to have it regardless, but it shouldn't be necessary if we didn't screw up above